File: TeamCode/src/main/java/teamcode/robot/core/RobotConfig.java
package teamcode.robot.core;

import com.bylazar.configurables.annotations.Configurable;

@Configurable
public class RobotConfig {
    public static double TurretPGain= 4;
    public static double TurretIGain= 0;
    public static double TurretDGain= 0.1;
    public static double TurretSpeedClamp =0.5;
    public static double TurretScaleFactor =0.01;
    public static double ShooterPower=0.8;
    public static double ShooterIdlePower=0.2;
    public static double TurnScale=0.5;
    public static double PowerScale=0.5;
}

File: TeamCode/src/main/java/teamcode/robot/core/RobotConfig.kt
package teamcode.robot.core

import com.bylazar.configurables.annotations.Configurable

@Configurable
object RobotConfig {
    var TurretPGain: Double = 4.0
    var TurretIGain: Double = 0.0
    var TurretDGain: Double = 0.1
    var TurretSpeedClamp: Double = 0.5
    var TurretScaleFactor: Double = 0.01
    var ShooterPower: Double = 0.8
    var ShooterIdlePower: Double = 0.2
    var TurnScale: Double = 0.5
    var PowerScale: Double = 0.5
}
